documentation/whats-new.md

```diff
@@ -1,10 +1,18 @@
 <!--
- Copyright 2020 Google LLC
+ Copyright 2020-2021 Google LLC
  Use of this source code is governed by an MIT-style
  license that can be found in the LICENSE file or at
  https://opensource.org/licenses/MIT.
 -->
 
+> Subscribe to [mystudies-announce@googlegroups.com](https://groups.google.com/g/mystudies-announce/) to receive release notifications and announcements
+
```

Comment:
Please add # Release 2.0.2 to this file

Reply:
Added # Release 2.0.2 summary in `whats-new.md` file.

Reply:
Nit - capitalize "Auth Server" so that it matches the other component names. Also did you want to link to the issues that were resolved in this release? Is there an elegant way to link to the milestone? @SR-btc

Reply:
We can hyperlink # Release 2.0.2 to https://github.com/GoogleCloudPlatform/fda-mystudies/milestone/5

Please confirm so @harisboston can make both these edits

Reply:
That is a good link to include. I would add the link in its own bullet (rather than making the title a link), for example: "The full list of bugs and features addressed in this release can be viewed [here]"

Reply:
@jkrogers18 Updated [whats-new](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/2.0.2-release-candidate/documentation/whats-new.md) file as per above comments

Reply:
I just submitted a commit that fixes the bullet and link